CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ need):
 
     ./configure --enable-extrachecks --enable-asan
 
-### enable-extrachecks
+#### enable-extrachecks
```

Comment:
I updated the heading level of this section because it looks like it is sub-information to the prior section.